app/_hub/kong-inc/application-registration/overview/_index.md

```diff
@@ -5,7 +5,10 @@ nav_title: Overview
 Applications allow registered developers on Kong Developer Portal to
 authenticate against a Gateway service. Dev Portal admins can
 selectively admit access to services using the
-Application Registration plugin.
+Application Registration plugin. Furthermore, enabling `enable_proxy_with_consumer_credential` will introduce
+an additional method for accessing the service using the consumer's credential.
+It is essential to exercise caution when enabling this parameter.
+Once activated, it will bypass the plugin's ACL control, potentially compromising security measures.
```

Comment:
```suggestion
Application Registration plugin. 

{:.note}
> **Note**: By enabling the `application-registration` plugin on a service, access to the service with consumer credentials will be disallowed by default. To allow this behavior, please refer to [these instructions](/hub/kong-inc/application-registration/how-to/#allow-accessing-the-service-with-consumer-credentials-optional) for advanced configuration.
```

As this is optional for our users, would it be better for us to leave a link for them here?

_For Docs team reviewers_: Could you help check if the anchor in the link above is correct? Thanks!